doc/reference/configuration/configuration_reference.rst

```diff
@@ -2400,7 +2441,12 @@ The ``process`` defines configuration parameters of the Tarantool process in the
 
 .. confval:: process.background
 
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
     Run the server as a background task.
+    The :ref:`process.pid_file <configuration_reference_process_pid_file>`
+    and :ref:`log <configuration_reference_log>`
+    parameters must be non-null for this to work.
```

Comment:
This sentence is correct in context of `box.cfg` options, but it is a bit imprecise in context of the options from the declarative configuration. Let's look on them separately.

`process.pid_file` has a default value (`var/run/{{ instance_name }}/tarantool.pid`). A missing value or `null` means that the effective value is the default value. We can assume that the PID file is always created and consider this requirement as always met. There is no reason to highlight it to a user.

If you see the opposite behavior, I would like to know details, because it is not what I expect.

`log` is a section and its options have defaults. It means that the effective `log` value can't be `null`.

However, that's true that we should configure logs to `file`, `pipe` or `syslog` (anything other than `stderr`). The `log.to` option is for that. Its default is `stderr` and it actually requires an attention from a user, when `process.background` is set to `true`.

The reason why logging to `stderr` can't be used with a daemonization is that a daemon process is detached from a terminal and it can't write to the terminal's stderr. See also [man 3 daemon](https://man7.org/linux/man-pages/man3/daemon.3.html)[^1].

[^1]: Tarantool implements this functionality by itself, but the idea is the same. We close terminal's stdin/stdout/stderr too.